sig-big-data/resources.md

```diff
@@ -1,15 +1,61 @@
 # Resources
 
+## Kubernetes Integration status by Big Data product
```

Comment:
Can we use the capital case for the first letter of each word consistently?

Reply:
@liyinan926 pushed change in latest commit. looks consistent now?